### ISSUE TITLE
libuuid is broken on Haiku?

### ISSUE THREAD

**Issue by Begasus** [haikuports/haikuports#4726]
After recent update to libuuid (https://github.com/haikuports/haikuports/commit/ce55d8052b51c8e953a5a35b2b4f87cfd3592e12) it doesn't build anymore (checked 32bit (nightly and beta))

```
waiting for build package libuuid_x86-1.3.1-1 to be activated
waiting for build package libuuid_x86_devel-1.3.1-1 to be activated
Building ...
scons: Reading SConscript files ...
Checking for C function usleep()... no
Checking for C function nanosleep()... no
Checking for C header file unistd.h... no
Checking for C header file stdlib.h... no
Checking for C header file sys/time.h... no
Checking for C header file sys/file.h... no
Checking for C header file sys/ioctl.h... no
Checking for C header file sys/un.h... no
Checking for C header file sys/sockio.h... no
Checking for C header file net/if.h... no
Checking for C header file netinet/in.h... no
Checking for C header file net/if_dl.h... no
Checking for C header file sys/syscall.h... no
scons: done reading SConscript files.
scons: Building targets ...
gcc -o src/clear.os -c -fPIC -Iinclude src/clear.c
gcc -o src/compare.os -c -fPIC -Iinclude src/compare.c
gcc -o src/copy.os -c -fPIC -Iinclude src/copy.c
gcc -o src/gen_uuid.os -c -fPIC -Iinclude src/gen_uuid.c
gcc -o src/isnull.os -c -fPIC -Iinclude src/isnull.c
gcc -o src/pack.os -c -fPIC -Iinclude src/pack.c
gcc -o src/parse.os -c -fPIC -Iinclude src/parse.c
scons: *** [src/clear.os] sh: No such file or directory: 'sh'
scons: *** [src/compare.os] sh: No such file or directory: 'sh'
scons: *** [src/copy.os] sh: No such file or directory: 'sh'
scons: *** [src/gen_uuid.os] sh: No such file or directory: 'sh'
scons: *** [src/isnull.os] sh: No such file or directory: 'sh'
scons: *** [src/pack.os] sh: No such file or directory: 'sh'
scons: *** [src/parse.os] sh: No such file or directory: 'sh'
gcc -o src/predefined.os -c -fPIC -Iinclude src/predefined.c
scons: *** [src/predefined.os] sh: No such file or directory: 'sh'
scons: building terminated because of errors.
Warning: Command '['bash', '-c', '. /wrapper-script']' returned non-zero exit status 2
Error: Build has failed - stopping.
```

**Comment by kallisti5** [comment]
Ack. Forgot about this one.  Korli mentioned it in that original commit.  I'll stand up a 32-bit machine and take a look.

**Comment by kallisti5** [comment]
Err... is SCons broken on x86_gcc2?

It reports running the "correct" gcc command.. then shows the "sh: No such file or directory: 'sh'"

**Comment by korli** [comment]
fixed in d033783342a95c422294a24a3bfc0adffb2540f8
